### DISCUSSION TITLE
Multiselect add to queue

### DISCUSSION THREAD

**Discussion by LukaszP2** [music-assistant/support#610] [Feature requests and ideas]
when I do multi select on list of tracks or artists or some where else it should be option: Add to queue

**Comment by marcelveldt** [comment]
This already exists....

For the list view: click the thumb image to select/deselect
For the panel view: press and hold to select

If items are selected: click the text in the header "x items selected" to open the contextmenu
The UX should be improved some day in the future but at least the functionality is there.

**Reply by marcelveldt** [reply]
<img width="486" alt="Schermafbeelding 2022-07-08 om 19 23 05" src="https://user-images.githubusercontent.com/6389780/178040688-1b4d7334-c501-4505-8671-18933f33187c.png">

**Reply by LukaszP2** [reply]
Yes, it should be :) but it's very good project! Some time it will be perfect 👌
Thank for your time!
